<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebitWindow</class>
 <widget class="QMainWindow" name="DebitWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
    background-color: qlineargradient(spread:pad, x1:0.46, y1:0, x2:0.472, y2:1, stop:0 rgba(0, 20, 15, 255), stop:1 rgba(17, 73, 74, 255));
    color: #c9bfb6;
}
QPushButton{
    background-color: qlineargradient(spread:pad, x1:0.494, y1:0, x2:0.517, y2:1, stop:0 rgba(36, 39, 43, 255), stop:1 rgba(72, 74, 78, 255));
    border:0px solid black;
    border-radius: 12px;
    color: #c9bfb6;
    font-size: 14px;
}
QLabel{
    background-color: transparent;
}
QLineEdit{
    border: 1px solid #c9bfb6;
    padding: 8px;
    color: #c9bfb6;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn20">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>240</width>
      <height>90</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>240</width>
      <height>90</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>240</width>
      <height>90</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>20 euros</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn40">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>240</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>40 euros</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn60">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>240</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>60 euros</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn100">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>140</y>
      <width>240</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100 euros</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn200">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>270</y>
      <width>240</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>200 euros</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn500">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>400</y>
      <width>240</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>500 euros</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtSelect">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select an amount</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string> Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_balance">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>0</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOther">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>530</y>
      <width>240</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Withdraw other amount</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>620</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_other">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>530</y>
      <width>240</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
